src/codemagic/tools/_app_store_connect/actions/publish_action.py

```diff
@@ -234,6 +242,7 @@ def _get_app_store_connect_submit_options(
                 support_url=support_url,
                 whats_new=whats_new.value if isinstance(whats_new, Types.WhatsNewArgument) else whats_new,
                 app_store_version_localizations=app_store_version_localizations,
+                cancel_previous_submissions=cancel_previous_submissions,
```

Comment:
I suggest to move it up after `max_build_processing_wait` as it does not belong under _Localized app metadata_ variables.